compat/mingw.c

```diff
@@ -3104,13 +3104,49 @@ int msc_startup(int argc, wchar_t **w_argv, wchar_t **w_env)
 
 #else
 
+static void maybe_redirect_std_handle(const wchar_t *key, DWORD std_id, int fd,
```

Comment:
Oops, of course! Too late now for the MinGit "side release" I am uploading right now for testing, but I will address this issue tomorrow.